src/ars/core/repository.py

```diff
@@ -22,7 +22,8 @@
 
 from ghga_service_commons.auth.ghga import AuthContext, has_role
 from ghga_service_commons.utils.utc_dates import now_as_utc
-from pydantic import BaseSettings, EmailStr, Field
+from pydantic import EmailStr, Field
```

Comment:
`EmailStr` is removed everywhere else.
Is this the sole entrypoint where this needs validation and further down this is simply handled as string or is this an oversight?

Reply:
No this can be removed here as well.